assets/components/svg/svg.jsx

```diff
@@ -644,7 +673,7 @@ function SvgCross() {
     <svg
       className="svg-cross"
       xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 36 24"
+      viewBox="-2 -2 38 38"
```

Comment:
Did you check this change didn't break the SVG in other places we use the fb logo?

Reply:
Yes it's fine.

Reply:
@svillafe I think this is actually the cross SVG. It's fine, this isn't used anywhere else that's in production.